api/geth_backend.go

```diff
@@ -315,6 +315,9 @@ func (b *GethStatusBackend) getAccountByKeyUID(keyUID string) (*multiaccounts.Ac
 	}
 	for _, acc := range as {
 		if acc.KeyUID == keyUID {
+			for k, v := range acc.Images {
+				acc.Images[k].LocalURL = b.statusNode.HTTPServer().MakeAccountImageURL(acc.KeyUID, v.Name, v.Clock)
```

Comment:
I think we should find a way to avoid duplicating this `.LocalURL = api.mediaServer.MakeAccountImageURL(keyUID, v.Name, v.Clock)` 3 times in the code...